### PR TITLE
Added version field for the mod authors to add

### LINKED ISSUE #21846: Version field in modinfo.json
Should we have some additional field showing mod version in `modinfo.json`?

### PR DESCRIPTION
**Pull request by Night-Pryanik**
Closes #21846

default
The description says it all.

### CODE REVIEW

File: src/mod_manager_ui.cpp [hunk 1]
```diff
@@ -88,6 +88,11 @@ std::string mod_ui::get_information( MOD_INFORMATION *mod )
              << "</color>: " << str << "\n";
     }
 
+    if( !mod->version.empty() ) {
+        info << "<color_light_blue>" << _( "Mod version" ) << "</color>: " <<
+             ( mod->version.c_str() ) << "\n";
```
Comment (codemime) [inline src/mod_manager_ui.cpp]: Can be boiled down to `<< mod->version << std::endl`. Also, formatting (`<< ` aligned vertically).

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Pull request merged**